src/catalog/src/builtin.rs

```diff
@@ -6765,6 +6766,14 @@ SELECT
         WHEN class_objects.type = 'view' THEN 'v'
         WHEN class_objects.type = 'materialized-view' THEN 'm'
     END relkind,
+    COALESCE(
```

Comment:
Needed to coalesce otherwise `show columns` would have this column show up as nullable